spec/requests/admin_claim_checks_spec.rb

```diff
@@ -47,11 +48,11 @@
       follow_redirect!
     end
 
-    describe "claim_approvals#create" do
+    describe "claim_checks#create" do
       let(:claim) { create(:claim, :submitted) }
 
       it "does not allow a claim to be approved" do
```

Comment:
I don't understand what this spec is testing. Why doesn't it allow a claim to be approved, and this doesn't appear to actually be testing attempting to approve it (it should have the status parameter, shouldn't it?